spec/models/attachment_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Attachment, type: :model do
```

Comment:
its better to add few more test cases for testing presence of attached file.

example:
expect(subject.attaching_file.attached?).to be_truthy

and add factory bot for easier attaching using traits.